src/tools/rbd_mirror/LeaderWatcher.h

```diff
@@ -17,6 +17,9 @@
 #include "MirrorStatusWatcher.h"
 #include "tools/rbd_mirror/leader_watcher/Types.h"
 
+#pragma GCC diagnostic push
```

Comment:
Ditto here?

Reply:
Same problem as before, the version in this class doesn't take the parameter that the base class does.